lib/langchain/tool/base.rb

```diff
@@ -72,6 +72,22 @@ def tool_description
       self.class.const_get(:DESCRIPTION)
     end
 
+    #
+    # Returns the SCHEMA constant of the tool
+    #
+    # @return [Hash] tool description
+    #
+    def tool_schema
```

Comment:
```suggestion
    def json_schema
```
wdyt?

Reply:
I don't like it that much, I think it's better to keep it consistent with the rest of the names. Maybe if your last PR gets approved this will be just `schema`